File: test/ws_cli.py
#!/usr/bin/env python

import asyncio
# pip install websockets
import websockets
import ssl
async def hello(url):
    cert_ignore = ssl._create_unverified_context() if url.startswith('wss') else None
    async with websockets.connect(url, ssl=cert_ignore) as websocket:
        msg = "Hello world! to " + url
        await websocket.send(msg)
        print(f">>> {msg}")
        strback = await websocket.recv()
        print(f"<<< {strback}")

asyncio.run(hello("ws://localhost:8888/ws"))
asyncio.run(hello("wss://localhost:9999/hello"))

File: test/ws_cli.js
#!/usr/bin/env node
import WebSocket from 'ws';
// npm install ws -S
// chmod +x ws_cli.{js,py}
const ws = new WebSocket('ws://localhost:8888/ws');

ws.on('open', function open() {
  ws.send('something');
});

ws.on('message', function message(data) {
  console.log('received: %s', data);
});
ws.on('error', (err)=> {
    console.log(`err: ${err}`);
});
ws.on('close', (code, reason)=> {
    console.log(`reason: ${reason}`);
});